build_docs.pl

```diff
@@ -294,7 +295,7 @@ sub check_links {
 
     $link_checker->check;
 
-    check_kibana_links( $build_dir, $link_checker );
+    check_kibana_links( $build_dir, $link_checker ) if exists $Conf->{repos}{kibana};
```

Comment:
I opened #614 to talk about how it is odd we have a specific repo here. We can do better, I think.